public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -1,702 +1,168 @@
-  But where?
-
-  We have to ***tell it where*** by adding `<router-outlet>` marker tags to the bottom of the template.
-  `RouterOutlet` is one of the `ROUTER_DIRECTIVES`.
-  The router displays each component immediately below the `<router-outlet>` as we navigate through the application.
-
-  ### Router Links
-  We don't really expect users to paste a route URL into the address bar.
-  We add an anchor tag to the template which, when clicked, triggers navigation to the `HeroesComponent`.
-
-  The revised template looks like this:
-+makeExample('toh-5/dart/lib/app_component_2.dart', 'template', 'lib/app_component.dart (template v1)')(format=".")
-:marked
-  Notice the `[routerLink]` binding in the anchor tag.
-  We bind the `RouterLink` directive (another of the `ROUTER_DIRECTIVES`) to a list
-  that tells the router where to navigate when the user clicks the link.
-
-  We define a *routing instruction* with a *link parameters list*.
-  The list only has one element in our little sample, the quoted ***name* of the route** to follow.
-  Looking back at the route configuration, we confirm that `'Heroes'` is the name of the route to the `HeroesComponent`.
-.l-sub-section
+block router-config-intro
   :marked
-      Learn about the *link parameters list* in the [Routing](../guide/router.html#link-parameters-array) chapter.
-:marked
-  Refresh the browser.  We see only the app title. We don't see the heroes list.
-.l-sub-section
-  :marked
-    The browser's address bar shows `/`.
-    The route path to `HeroesComponent` is `/heroes`, not `/`.
-    We don't have a route that matches the path `/`, so there is nothing to show.
-    That's something we'll want to fix.
-:marked
-  We click the "Heroes" navigation link, the browser bar updates to `/heroes`,
-  and now we see the list of heroes. We are navigating at last!
-
-  At this stage, our `AppComponent` looks like this.
-+makeExample('toh-5/dart/lib/app_component_2.dart',null, 'lib/app_component.dart (v2)')
-:marked
-  The  *AppComponent* is now attached to a router and displaying routed views.
-  For this reason and to distinguish it from other kinds of components,
-  we call this type of component a *Router Component*.
-
-
-:marked
-  ## Add a *Dashboard*
-  Routing only makes sense when we have multiple views. We need another view.
+    ### Configure routes and add the router
 
-  Create a placeholder `DashboardComponent` that gives us something to navigate to and from.
-+makeExample('toh-5/dart/lib/dashboard_component_1.dart',null, 'lib/dashboard_component.dart (v1)')(format=".")
-:marked
-  We’ll come back and make it more useful later.
+    The `AppComponent` doesn't have a router yet. We'll use the `@RouteConfig`
+    annotation to simultaneously:
 
-  ### Configure the dashboard route
-  Go back to `app_component.dart` and teach it to navigate to the dashboard.
+    - Assign a router to the component and
```

Comment:
Delete the "and"
(we don't treat list items like parts of a continuing sentence)